### DISCUSSION TITLE
The comment section is not set properly, and it will become blank after setting

### DISCUSSION THREAD

**Discussion by bainano** [GetPublii/Publii#1613] [Bug Report]
### Operating system

windows

### Publii version

new

### Issue type

Application

### Bug description

The comment section is not set properly, and it will become blank after setting

**Comment by bobmitro** [comment]
What theme do you use? Please provide the URL to the blank section.

**Comment by bainano** [comment]
是设置自定义html里的评论的代码，保存设置后不起作用，重新打开又变成空白，所有主题都会碰到这个问题
It's the code to set the comment in the custom html.,It doesn't work after saving the settings.,Reopen it and become blank.,All themes will run into this problem.
&nbsp;


mr.jin.
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Bob ***@***.***&gt;; 
发送时间: 2024年8月17日(星期六) 中午11:35
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [GetPublii/Publii] The comment section is not set properly, and it will become blank after setting (Discussion #1613)















你用什么主题的请提供空白部分的URL。

-
直接回复这封邮件，在GitHub上查看，或取消订阅.
***@***.***&gt;

**Reply by bobmitro** [reply]
There may be an issue with your code. Instead, I recommend using the [Comments ](https://marketplace.getpublii.com/plugins/) plugins.
